lib/tasks/subscriptions/debug.rake

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/BlockLength
+namespace :ofn do
+  namespace :subs do
+    namespace :debug do
+      desc "Print standard info about a specific Order Cycle"
+      task order_cycle: :environment do
+        order_cycle_id = request_order_cycle_id
+
+        order_cycle = OrderCycle.find_by(id: order_cycle_id)
+        puts "Order Cycle #{order_cycle.name}"
+        order_cycle.schedules.each do |schedule|
+          puts "Schedule #{schedule.name}"
+          Subscription.where(schedule_id: schedule.id).each do |subscription|
+            puts
+            puts "Subscription #{subscription.id}"
+            puts subscription.shop.name
+            puts subscription.customer.email
+            puts subscription.payment_method.name
+            puts "Active from #{subscription.begins_at} to #{subscription.ends_at}"
+            puts "Last edited on #{subscription.updated_at}"
+            puts "Canceled at #{subscription.canceled_at} and paused at #{subscription.paused_at}"
+
+            ProxyOrder.where(order_cycle_id: order_cycle_id,
+                             subscription_id: subscription.id).each do |proxy_order|
```

Comment:
We should stay on the safe side with all these DB fetches using [find_each](https://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each) and avoid potential problems in the future. I'm happy to go on with this and fix it in a separate PR myself.